build_envoy.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+mkdir -p build
```

Comment:
Would this file fit better in `scripts` directory? You would need to set the build dir differently then.

Reply:
@ipuustin 

> Would this file fit better in scripts directory?

OK, let's move it there.

 > You would need to set the build dir differently then.

Updated to "mkdir -p ${PWD}/build". Not sure if it makes more sense this way, please take a look and comment.

Reply:
The other script in the `scripts` dir does gets the directory paths this way:

```
SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
BASE_DIR="$(realpath "${SCRIPT_DIR}/..")"
```

Your command would be `mkdir -p "${BASE_DIR}/build"` then.

Reply:
@ipuustin 

Thanks. OK, added & updated.